server/api/users.js

```diff
@@ -32,3 +17,5 @@ router.get('/', requireToken, async (req, res, next) => {
     next(err);
   }
 });
+
+module.exports = { router };
```

Comment:
other files are exporting like this:
module.exports = router;

any specific reason why you chose to export router a bit different here?

Reply:
Same here. For some reason we were having troubles with it, so Lisa told me to do it like this as well. But happy to tinker with it if we need to.